app/api/route.js

```diff
@@ -0,0 +1,298 @@
+import Author from "@/models/authors";
+import { ApolloServer } from "@apollo/server";
+import { startServerAndCreateNextHandler } from "@as-integrations/next";
+import DataLoader from "dataloader";
+import { gql } from "graphql-tag";
+import moment from "moment/moment";
+import { validateImageUrl } from "../utils/imageValidator";
+const Book = require("../../models/books");
+const { Op } = require("sequelize");
+
+const typeDefs = gql`
+  type Book {
+    id: Int!
+    title: String!
+    description: String!
+    publishedAt: String!
+    profilePhotoUrl: String
+    author: Author!
```

Comment:
Just headsup [In week 4] when you do this part of assignment again, remember that Book can have multiple Authors.